src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts

```diff
@@ -168,6 +169,7 @@ export class UBSOrderDetailsComponent implements OnInit, OnDestroy {
         this.showCertificateUsed = this.total;
         this.points = this.orders.points + this.certificateLeft;
       }
+      this.bonusesRemaining = this.certificateSum > 0;
```

Comment:
we have the same logic in Html file as well - certificateSum > 0
so let's move it to a separate getter or method